### DISCUSSION TITLE
Как исправить расположение видимости репозитория

### DISCUSSION THREAD

**Discussion by NatashkaJK** [community/community#121214] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Вводя git status , у меня репозиторий видит весь мой диск, вместо определенной папки с репозиторием, что делать?
![image](https://github.com/community/community/assets/158469400/17876e13-4407-401e-84c4-edab4fb8a8c3)
Я пыталась удалить, но пишет, что не хватает доступа для вхождения в папки
![image](https://github.com/community/community/assets/158469400/02eca8e7-22da-4cf6-a949-fc7a1619767d)

**Comment by Akash1134** [comment]
Hi @NatashkaJK , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
